alphastats/gui/pages/06_Results.py

```diff
@@ -21,9 +21,11 @@
 
     method = saved_item[0]
```

Comment:
```suggestion
for count, (method, plot, parameters) in enumerate(st.session_state[StateKeys.PLOT_LIST]):
```
we could change the loop to this, or is it making this less readable?

Reply:
in principle, yes, but then the `.remove(saved_item)` would become less readable